src/bin/pgcopydb/cli_common.c

```diff
@@ -212,104 +212,64 @@ cli_copydb_getenv(CopyDBOptions *options)
 {
 	int errors = 0;
 
-	if (!cli_copydb_getenv_source_pguri(&(options->connStrings.source_pguri)))
-	{
-		/* errors have already been logged */
-		++errors;
-	}
+	/* Fill in the defaults before reading environment variables */
+	options->tableJobs = DEFAULT_TABLE_JOBS;
+	options->indexJobs = DEFAULT_INDEX_JOBS;
+	options->restoreOptions.jobs = DEFAULT_RESTORE_JOBS;
+	options->lObjectJobs = DEFAULT_LARGE_OBJECTS_JOBS;
+	options->splitTablesLargerThan.bytes = DEFAULT_SPLIT_TABLES_LARGER_THAN;
+
+	EnvParser parsers[] = {
+		{ PGCOPYDB_TABLE_JOBS, ENV_TYPE_INT,
+		  &(options->tableJobs), 0, 1, 1, 1, 128 },
```

Comment:
I would use `true` instead of 1 for the boolean value for the “bounded” slot of the structure here.